client/lib/deterministic-stringify/index.js

```diff
@@ -1,32 +1,39 @@
-function deterministStringify( source ) {
-	var namespace = [],
-		keys;
+export function deterministicStringify( source ) {
+	if ( source === null ) {
+		return 'null';
+	}
+
+	// Handle primitive data types:
+	// boolean, number, string, undefined
 	if ( typeof source !== 'object' ) {
 		if ( typeof source === 'undefined' ) {
 			return 'undefined';
 		}
+
 		if ( typeof source === 'string' ) {
-			return "'" + source.toString() + "'";
+			return `'${ source.toString() }'`;
```

Comment:
`'${ source }'` would do exactly the same job. There's a nice article that explains all the shady details of string conversion here: http://2ality.com/2012/03/converting-to-string.html